ipaserver/install/ipa_backup.py

```diff
@@ -400,7 +400,7 @@ def get_connection(self):
             self._conn.external_bind()
         except Exception as e:
             logger.error("Unable to bind to LDAP server %s: %s",
-                         self._conn.host, e)
+                         self._conn.ldap_uri, e)
 
```

Comment:
Having the full uri might be handy because it includes the protocol used so one can distinguish a network (ldap/ldaps) connection vs a local socket (ldapi).